### ISSUE TITLE
Signing certificate issues

### ISSUE THREAD

**Issue by SergeWilfried** [documenso/documenso#680]
### Issue Description

Hey team. Thanks again for the incredible work with Documenso. I am getting the following error trying to sign a document on singleplayer mode with a self signed certificate.

```
⨯ ../node_modules/node-forge/lib/pkcs12.js (475:12) @ Object.p12.pkcs12FromAsn1
@documenso/marketing:dev:  ⨯ Error: PKCS#12 MAC could not be verified. Invalid password?
@documenso/marketing:dev:     at signWithLocalCert (../../packages/signing/transports/local-cert.ts:25:68)
@documenso/marketing:dev:     at signPdf (../../packages/signing/index.ts:11:12)
@documenso/marketing:dev:     at createSinglePlayerDocument (single-player-mode/create-single-player-document.action.ts:100:29)
@documenso/marketing:dev: null
```
I have updated the password a couple of times to make sure that there was no typo. But it does not work.

### Steps to Reproduce

- [x] Clone Repo
- [x] Follow development Instructions
- [x] Create Certificate
- [x] Launch App
- [x] Go to http://localhost:3001/singleplayer
- [x] Sign a document
- [x] See Error

### Operating System [e.g., Windows 10]

Mac Os 12.3.1

### Browser [e.g., Chrome, Firefox]

Chrome

### Please check the boxes that apply to this issue report.

- [X] I have searched the existing issues to make sure this is not a duplicate.
- [X] I have provided steps to reproduce the issue.
- [X] I have included relevant environment information.
- [X] I have included any relevant screenshots.
- [X] I understand that this is a voluntary contribution and that there is no guarantee of resolution.
- [ ] I want to work on creating a PR for this issue if approved

**Comment by crediblebilal** [comment]
This error occurs in [Documenso Website](https://documenso.com/singleplayer/) in the **Singleplayer** mode as well. After signing the document, we are redirected to `/singleplayer/undefined/success`, i.e we are getting **undefined** instead of the expected **documentToken**.

**Comment by Mythie** [comment]
> This error occurs in [Documenso Website](https://documenso.com/singleplayer/) in the **Singleplayer** mode as well. After signing the document, we are redirected to `/singleplayer/undefined/success`, i.e we are getting **undefined** instead of the expected **documentToken**.

This sounds super grim! I’m onto it 😰

**Comment by crediblebilal** [comment]
What I observed:

1.  In local setup, if I choose fields other than Signature i.e Name, Email or Date, I get error in sending email after signing the document which is quite obvious as I have not set it up properly. I skipped the **sendMail** part and the issue was resolved. But check it in production because signer does not receive any email after signing the document.
2. In local setup, if I choose Signature field, I get the **Error: Unknown font format** error as well. Check it in prod.
